usr/src/uts/common/fs/zfs/zap.c

```diff
@@ -292,9 +294,11 @@ zap_table_load(zap_t *zap, zap_table_phys_t *tbl, uint64_t idx, uint64_t *valp)
 		 */
 		blk = (idx*2) >> (bs-3);
 
-		err = dmu_buf_hold(zap->zap_objset, zap->zap_object,
+		dnode_t *dn = dmu_buf_get_dnode(zap->zap_dbuf);
```

Comment:
`dn` declared twice in same function. Would be clearer to use different names or declare once in outer scope.